libass/ass_outline.c

```diff
@@ -323,6 +323,21 @@ bool outline_transform_3d(ASS_Outline *outline, const ASS_Outline *source,
     return true;
 }
 
+int32_t outline_find_min_transformed_x(const ASS_Outline *outline,
```

Comment:
I think it's more clear to add `int32_t *leftmost_x` as a parameter and update current minimum instead of overwriting it.